pom.xml

```diff
@@ -42,11 +42,12 @@
 	</modules>
 
 	<properties>
-		<openMRSVersion>2.1.6</openMRSVersion>
-		<openMRSWebServicesVersion>2.24.0</openMRSWebServicesVersion>
+		<openMRSVersion>2.4.2</openMRSVersion>
+		<openMRSWebServicesVersion>2.29.0</openMRSWebServicesVersion>
```

Comment:
this should be  bumped to. 2.33.0, or whatever we are using for https://github.com/Bahmni/openmrs-distro-bahmni/blob/master/distro/pom.xml#L46 (Notice there is no branch for 2.4.x or 2.5)

Reply:
Angshu,
we were going through 2.4.0 release notes and there they have mentioned web-services version 2.29.0, we are planning to 2.30.0 for 2.5.0 OMRS upgrade
https://wiki.openmrs.org/display/RES/Platform+Release+Notes+2.4.0